python/test/t_Normal_std.expout

```diff
@@ -1,5 +1,6 @@
-Standard distribution =  Normal(mu = 0, sigma = 1)
-Get roughness =  0.28209
+Standard distribution =  Normal(mu = 0, sigma = 1) roughness =  0.28209
+2D distribution =  class=Normal name=Normal dimension=2 mean=class=Point name=Unnamed dimension=2 values=[0,0] sigma=class=Point name=Unnamed dimension=2 values=[1,1] correlationMatrix=class=CorrelationMatrix dimension=2 implementation=class=MatrixImplementation name=Unnamed rows=2 columns=2 values=[1,0,0,1] roughness =  0.07958
+2D with correlation distribution =  class=Normal name=Normal dimension=2 mean=class=Point name=Unnamed dimension=2 values=[0,0] sigma=class=Point name=Unnamed dimension=2 values=[1,2] correlationMatrix=class=CorrelationMatrix dimension=2 implementation=class=MatrixImplementation name=Unnamed rows=2 columns=2 values=[1,0.3,0.3,1] roughness =  0.04171
```

Comment:
Where do these reference values come from?

Reply:
For standard distribution, litterature. 
I implemented an estimate using simulation (similar to your code)